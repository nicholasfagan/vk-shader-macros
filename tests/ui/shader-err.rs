use vk_shader_macros::compile_glsl;

const SHADER: &'static [u32] = compile_glsl!(r##"
#version 450
void main() {
    nonsense;
	gl_Position = vec4(0,0,0,0);
}
"##, kind: vert);

fn main() {}