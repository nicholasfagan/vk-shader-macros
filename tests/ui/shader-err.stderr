error: compilation error:
<inline#0 bytes(82..163)>:4: error: 'nonsense' : undeclared identifier

 --> $DIR/shader-err.rs:3:46
  |
3 |   const SHADER: &'static [u32] = compile_glsl!(r##"
  |  ______________________________________________^
4 | | #version 450
5 | | void main() {
6 | |     nonsense;
7 | |     gl_Position = vec4(0,0,0,0);
8 | | }
9 | | "##, kind: vert);
  | |___^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
